src/main/java/FormController.java

```diff
@@ -8,21 +7,26 @@
 
 public class FormController implements Controller {
     private String resourcePath;
+    private String[] methodsAllowed;
+    private final String CARRIAGE_RETURN_NEWLINE = "\r\n";
```

Comment:
You can use System.getProperty("user.dir") instead of "\r\n" to make sure it uses the system Newline. Then you can rename it simply NEWLINE,